pode.build.ps1

```diff
@@ -501,15 +501,51 @@ Task TestNoBuild TestDeps, {
     else {
         $Script:TestStatus = Invoke-Pester -Configuration $configuration
     }
-    if ($originalUICulture){
+    if ($originalUICulture) {
         Write-Output "Restore UICulture to $originalUICulture"
         # restore original UICulture
         [System.Threading.Thread]::CurrentThread.CurrentUICulture = $originalUICulture
     }
 }, PushCodeCoverage, CheckFailedTests
 
-# Synopsis: Run tests after a build
-Task Test Build, TestNoBuild
+# Synopsis: Run tests after a build if needed
+Task Test {
```

Comment:
But there's already a `TestNoBuild` to do this, and skip the building if not needed? `Test` should always build, otherwise it won't be obvious this isn't occurring to people; if they run `Test`, change the .NET code, and run `Test` again, it wouldn't rebuild and use the new code.

If people want to run the tests without building, `TestNoBuild` is available.

Reply:
I forgot:(